simulariumio/data_objects/agent_data.py

```diff
@@ -35,10 +35,10 @@ class AgentData:
     positions: np.ndarray
     radii: np.ndarray
     rotations: np.ndarray
-    n_subpoints: np.ndarray = None
-    subpoints: np.ndarray = None
-    display_data: Dict[str, DisplayData] = {}
-    draw_fiber_points: bool = False
+    n_subpoints: np.ndarray
+    subpoints: np.ndarray
+    display_data: Dict[str, DisplayData]
+    draw_fiber_points: bool
```

Comment:
is this change related?

Reply:
no, just clean up, they're all set in the constructor